.github/workflows/post-promote-tasks.yml

```diff
@@ -0,0 +1,65 @@
+name: Post promote tasks
+
+on:
+  pull_request:
+    types:
+      - closed
```

Comment:
Wouldn't `closed` get triggered also when someone closes a PR without merging it?

Reply:
Yup, but there is no `merged` event type. There is an `if: merged" down below

Reply:
Oh, missed that